sdk/include/reactos/shlwapi_undoc.h

```diff
@@ -21,6 +21,11 @@
 #ifndef __SHLWAPI_UNDOC_H
 #define __SHLWAPI_UNDOC_H
 
+#include <unknwn.h>
+#include <docobj.h>
+#include <shlobj.h>
+#include <shobjidl.h>
```

Comment:
Why all those headers? They should be included instead in the module that also includes shlwapi_undoc.h

Reply:
Reverted.